RLTSquare/Training/Controller/Index/Indexes.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+namespace RLTSquare\Training\Controller\Index;
+
+use Magento\Framework\App\ActionInterface;
+use Magento\Framework\App\Request\Http;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Framework\View\Result\PageFactory;
+use Magento\Store\Model\StoreManagerInterface;
+use RLTSquare\Training\Logger\Logger;
+use Magento\Framework\Translate\Inline\StateInterface;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Framework\App\Area;
+
+class Indexes implements ActionInterface
+{
+    protected $emailTemplate;
+    protected $pageFactory;
+    /**
+     * @var Http
+     */
+    protected $request;
+    /**
+     * @var TransportBuilder
+     */
+    protected $transportBuilder;
+    /**
+     * @var StoreManagerInterface
+     */
+    protected $_storeManager;
+    private $logger;
+
+    public function __construct(
+        PageFactory $pageFactory,
+        Logger $logger,
+        Http $request,
+        TransportBuilder $transportBuilder,
+        StoreManagerInterface $storeManager,
+        StateInterface        $state,
+        \Magento\Email\Model\BackendTemplate $emailTemplate
+    ) {
+        $this->pageFactory = $pageFactory;
+        $this->logger = $logger;
+        $this->request = $request;
+        $this->transportBuilder = $transportBuilder;
+        $this->storeManager = $storeManager;
+        $this->inlineTranslation = $state;
+        $this->emailTemplate = $emailTemplate;
+    }
+
+    public function execute()
+    {
+        $this->sendEmail();
+        $this->logger->info('Page Visited');
+        $this->logger->info('Email Send');
+        $pageFactory = $this->pageFactory->create();
+        $pageFactory->getConfig()->getTitle()->set(__('Test'));
+        return $pageFactory;
+    }
+    public function sendEmail()
+    {
+        $emails_template = $this->emailTemplate->load('email_template', 'orig_template_code');
```

Comment:
Overlooked logic - the logic here is faulty. The requirement was to email template that user can edit and change from admin. That implies use of system.xml, with field specific for juggling email_template, which in turn gets set to default value of coded email template using config.xml.

```
               <field id="email_template" translate="label comment" type="select" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
                    <label>Email Template</label>
                    <comment>Email template chosen based on theme fallback when "Default" option is selected.</comment>
                    <source_model>Magento\Config\Model\Config\Source\Email\Template</source_model>
                </field>
```

Then, a helper method can be used for reading the template

```
    public function getEmailTemplate(): string
    {
        return (string)$this->scopeConfig->getValue(
            self::XML_PATH_EMAIL_TEMPLATE,
            ScopeInterface::SCOPE_STORE
        );
    }
```

```
            $transport = $this->transportBuilder
                ->setTemplateIdentifier($this->helper->getEmailTemplate())
```

this way we ensure either default meail template is used (through cdefaults in config.xml) or a reassigned one is used ... all through that one helper call